src/themes/teams/components/Avatar/avatarVariables.ts

```diff
@@ -1,4 +1,7 @@
 export default () => ({
-  statusBorderColor: 'white',
-  statusBorderWidth: 2,
+  label: { padding: '0px' },
+  status: {
```

Comment:
I would rather suggest to refrain from utilizing this pattern of nested (child) variables for now, as we are not completely sure if it is a way to go for us (couple of pending prototypes are under review currently). It seems that we might consider to hardcode these styles for now (in `avatarStyles` file) - although I do understand that we will lose a bit in terms of customization capabilities for the examples, at the same time we 
- won't restrict the functionality for Teams as a consumer
- would be at much safer spot in terms of using only the approaches that we have agreed upon

Reply:
Okay, will try to replace then all the variables that are referring some components slots with the slot's inside the styles.

Reply:
I reverted the changes I introduced for the status slot, and removed the label's variables. I wasn't able to get rid of the variables for the status, as these variables are used in multiple places for different calculation